Sources/Formworks/FWComponents/FWMultilineComponent/FWMultilineComponentView.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  FWMultilineComponentView.swift
+// 
+//
+//  Created by Rafael Galdino on 23/11/20.
+//
+
+import UIKit
+
+final class FWMultilineComponentView: UITableViewCell, FWComponentCell {
```

Comment:
Should this class be documented?

Reply:
Yes, it should. Unfortunately I couldn't afford to document this before merging due to our deadlines.
I'm creating the issue #98 with all the classes I missed documentation and will work on it as soon as possible.